src/data-handling/Trip.js

```diff
@@ -1 +1,70 @@
-export default class Trip {};
\ No newline at end of file
+import moment from 'moment';
+moment().format();
+
+export default class Trip {
+  constructor(data) {
+    this.tripId = data.id;
+    this.userId = data.userID;
+    this.destinationId = data.destinationID;
+    this.travelers = data.travelers;
+    this.startDate = data.date;
+    this.endDate;
+    this.duration = data.duration;
+    this.status = data.status;
+    this.suggestedActivities = data.suggestedActivities;
+  };
+
+  findCorrespondingTraveler(list) {    
+    return list.find(traveler => traveler.id === this.userId);    
+  }
+
+  findCorrespondingDestination(list) {
+    return list.find(destination => destination.id === this.destinationId);
+  }
+
+  returnBeginning() {    
+    return this.startDate;
+  }
+
+  returnDuration() {
+    return this.duration;
+  }
+
+  returnEnd() {
+    const format = 'YYYY/MM/DD';    
+    const startDate = moment(this.startDate, format);    
+    const endDate = startDate.add(this.duration - 1, "day");    
```

Comment:
I love how obvious this is with 'moment'. Do you need to add a set number of days? Then use "add". Okay, I'm sold.